routes/user.js

```diff
@@ -182,4 +218,34 @@ router.route("/changePassword/:id")
         });
     });
 
+router.route("/approve/:approvalS")
+    .all(function (req, res) {
+        var makeLink = function (length) {
+            var result = '';
+            var characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+            var charactersLength = characters.length;
+            for (var i = 0; i < length; i++) {
+                result += characters.charAt(Math.floor(Math.random() * charactersLength));
+            }
```

Comment:
This can be done smarter, look at Math.random().toString(36) (would also be a lot shorter)

Reply:
Can do that at a later stage if we want too